canonical/tempo-k8s-operator
# TLS integration for tempo endpoints and server

TLS integration support for Tempo charm

In principle ready for review, but it depends on ingress #94 

Testing instructions:


```
juju add-model test-tempo-tls
juju deploy cos-lite --trust
charmcraft pack
jhack deploy
jhack imatrix fill
```

you should be able to see charm traces in grafana for all the charms.
verify that all receiver endpoints are using https:

```
juju run tempo/0 list-receivers
```
